src/actions.py

```diff
@@ -68,12 +68,12 @@
     "escaped quad": ['\\"', '\\"']
 }
 
-mod.list("cursorless_wrapable", desc="Supported wraps for cursorless wrap action")
-ctx.lists["self.cursorless_wrapable"] = wrapables.keys()
+mod.list("cursorless_wrappers", desc="Supported wraps for cursorless wrap action")
```

Comment:
**nit**: (and sorry if this one was my fault) talon lists / captures are usually singular I believe
```suggestion
mod.list("cursorless_wrapper", desc="Supported wrappers for cursorless wrap action")
```